-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-42864/36999

ERROR: Build errors found during clang-tidy run.
```
DQM/L1TMonitor/src/L1TStage2EMTF.cc:769:3: error: use of undeclared identifier 'emtfMuonhwPtUnconstrained'; did you mean 'emtfMuonhwPtUnconstrinaed'? [clang-diagnostic-error]
  emtfMuonhwPtUnconstrained = 
  ^~~~~~~~~~~~~~~~~~~~~~~~~
  emtfMuonhwPtUnconstrinaed
--
DQM/L1TMonitor/src/L1TStage2EMTF.cc:771:3: error: use of undeclared identifier 'emtfMuonhwPtUnconstrained'; did you mean 'emtfMuonhwPtUnconstrinaed'? [clang-diagnostic-error]
  emtfMuonhwPtUnconstrained->setAxisTitle("Hardware Unconstrained p_{T}", 1);
  ^~~~~~~~~~~~~~~~~~~~~~~~~
  emtfMuonhwPtUnconstrinaed
--
DQM/L1TMonitor/src/L1TStage2EMTF.cc:1264:7: error: use of undeclared identifier 'emtfMuonhwPtUnconstrained'; did you mean 'emtfMuonhwPtUnconstrinaed'? [clang-diagnostic-error]
      emtfMuonhwPtUnconstrained->Fill(Muon->hwPtUnconstrained());
      ^~~~~~~~~~~~~~~~~~~~~~~~~
      emtfMuonhwPtUnconstrinaed
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/DQM/L1TMonitor/interface/L1TStage2EMTF.h
+++ b/DQM/L1TMonitor/interface/L1TStage2EMTF.h
@@ -77,7 +77,7 @@ private:
 
   MonitorElement* emtfMuonBX;
   MonitorElement* emtfMuonhwPt;
-  MonitorElement* emtfMuonhwPtUnconstrinaed;
+  MonitorElement* emtfMuonhwPtUnconstrained;
   MonitorElement* emtfMuonhwDxy;
   MonitorElement* emtfMuonhwEta;
   MonitorElement* emtfMuonhwPhi;

